extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -162,8 +138,8 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			return;
 		}
 
-		if (!isBlocking()) {
-			slotsToRecover.push(slotName);
+		if (!recoveryHelper.isBlocking()) {
```

Comment:
`!recoveryHelper.isBlocking() || !recoveryHelper. isRecoveryEnabled()`, otherwise it will have an impact to users from `wgAdDriverSorucePointDetection` countries. In this case it's not so dangerous but still we don't need to add this slots when recovery is disabled.